docs/user-guide/sans-polarization-analysis-methodology.ipynb

```diff
@@ -5,23 +5,20 @@
    "id": "788a4fce-63dc-4c23-8d2a-e9a05db0fdf7",
    "metadata": {},
    "source": [
-    "## Data correction procedure for TOF-SANS with polarization analysis\n",
+    "# SANS Polarization Analysis Methodology\n",
     "\n",
-    "- Note on notation:\n",
-    "> - subscripts = discretization variable from measurement\n",
-    "> - superscripts = Names\n",
-    "> - () = continuous function of....\n",
+    "## Notation\n",
+    "\n",
+    "Note on notation:\n",
+    "\n",
+    "- subscripts = discretization variable from measurement\n",
+    "- superscripts = Names\n",
+    "- () = continuous function of....\n",
     "\n",
     "| Value | Meaning | shape | How acquired? | \n",
     "|:--------:|:--------:|:--------:|:--------:|\n",
-    "|  $I^{++}_{t,\\lambda}$   |  uncorrected sample data of ++ channel |  data array with columns for $I$, $\\lambda$, $t$   |  neutron data |\n",
-    "|  $I^{+-}_{t,\\lambda}$   |  uncorrected sample data of +- channel |  data array with columns for $I$, $\\lambda$, $t$   | neutron data    |\n",
-    "|  $I^{-+}_{t,\\lambda}$   |  uncorrected sample data of -+ channel  |  data array with columns for $I$, $\\lambda$, $t$   | neutron data   |\n",
-    "|  $I^{--}_{t,\\lambda}$   |  uncorrected sample data of -- channel |  data array with columns for $I$, $\\lambda$, $t$    | neutron data   |\n",
-    "|  $S^{++}_{t,\\lambda}$   |  corrected sample data of ++ channel  |  data array with columns for $I$, $\\lambda$, $t$   | by pol-correction  |\n",
-    "|  $S^{+-}_{t,\\lambda}$   |  corrected sample data of +- channel  |  data array with columns for $I$, $\\lambda$, $t$   | by pol-correction   |\n",
-    "|  $S^{-+}_{t,\\lambda}$   |  corrected sample data of -+ channel  |  data array with columns for $I$, $\\lambda$, $t$   | by pol-correction   |\n",
-    "|  $S^{--}_{t,\\lambda}$   |  corrected sample data of -- channel  |  data array with columns for $I$, $\\lambda$, $t$   | by pol-correction    |\n",
+    "|  $I^{\\pm\\pm}_{t,\\lambda}$   |  uncorrected sample data of ++/+-/-+/-- channels |  data arrays with columns for $I$, $\\lambda$, $t$   |  neutron data |\n",
+    "|  $S^{\\pm\\pm}_{t,\\lambda}$   |  corrected sample data of ++/+-/-+/-- channels  |  data arrays with columns for $I$, $\\lambda$, $t$   | by pol-correction  |\n",
     "|  $p^{cell}$   |  pressure of <sup>3</sup>He-polarizer/analyzer  |  scalar, 1-dim   | from metadata  |\n",
     "|  $l^{cell}$   |  length of <sup>3</sup>He-polarizer/analyzer   |  scalar, 1-dim   | from metadata   |\n",
     "|  $T^{g, cell}(\\lambda)$   |  transmission of empty glass of polarizer/analyzer   |  scalar, 1-dim   | from metadata  |\n",
```

Comment:
If we want to be really picky, I think the commonly accepted convention is that anything in italics is a variable (i.e. something that can have different values), and any other sub/super-scripts should be in normal font.
So I guess `g` should not be in italics here as it codes for "glass"?

Same for `DB`, `3He`, etc. below.

But it's not that important, as I said, just being picky.

Reply:
There are many instances of this, and yes, I think it should be fixed, but it is unrelated to this PR.

Reply:
As this is your convention, of course yes! i will change it in the other PR / updated workflows I am writing for the instrument-specific details (e.g. where some data has to be acquired in a different way due to non-movable cells, samples, etc., and also the usage of supermirrors).

Reply:
> As this is your convention

I think this is pretty much standard in all scientific literature, right, i.e., not our standard?

Reply:
You are right